tensorboard/summary_test.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""API tests for the `tensorboard.summary` module.
+
+These tests are especially important because this module is the standard
+public entry point for end users, so we should be as careful as possible
+to ensure that we export the right things.
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import collections
+
+import tensorflow as tf
+from tensorboard import summary
+
+
+STANDARD_PLUGINS = frozenset([
+    'audio',
+    'histogram',
+    'image',
+    'pr_curve',
+    'scalar',
+    'text',
+])
+
+# The subset of `STANDARD_PLUGINS` for which we do not currently have
+# functions to generate a summary protobuf outside of a TensorFlow
+# graph. This set should ideally be empty; any entries here should be
+# considered temporary.
+PLUGINS_WITHOUT_PB_FUNCTIONS = frozenset([
+    'pr_curve',  # TODO(@chihuahua): Fix this.
```

Comment:
Nit: Instead of making this a raw TODO, can we link it to a GitHub issue?

Reply:
Sure.

Reply:
Added reference to #445, which already exists and is assigned to @chihuahua . Will merge once Travis finishes.